src/pages/dashboard/index.tsx

```diff
@@ -0,0 +1,244 @@
+                      <DescriptionList
+                        displayMode={DescriptionListDisplayMode.LABEL_ABOVE}
+                      >
+                        <DescriptionList.Item
+                          label={
+                            <div className="text-cool-gray-500 truncate text-sm font-medium leading-5">
+                              {t("Datasources")}
+                            </div>
+                          }
+                        >
+                          <div className="text-cool-gray-900 text-lg font-medium leading-7">
+                            {data?.catalog.totalItems}
+                          </div>
+                        </DescriptionList.Item>
+                      </DescriptionList>
+                    </div>
+                  </div>
+                </div>
+                <div className="bg-cool-gray-50 px-5 py-3">
+                  <div className="text-sm leading-5">
+                    <Link
+                      href="/catalog"
+                      className="font-medium text-blue-600 transition duration-150 ease-in-out hover:text-blue-900"
+                    >
+                      {t("View all")}
+                    </Link>
+                  </div>
+                </div>
+              </Block>
+              <Block>
+                <div className="p-5">
+                  <div className="flex items-center">
+                    <div className="flex-shrink-0">
+                      <BookOpenIcon className="text-cool-gray-400 h-6 w-6" />
+                    </div>
+                    <div className="ml-5 w-0 flex-1">
+                      <DescriptionList
+                        displayMode={DescriptionListDisplayMode.LABEL_ABOVE}
+                      >
+                        <DescriptionList.Item
+                          label={
+                            <div className="text-cool-gray-500 truncate text-sm font-medium leading-5">
+                              {t("Notebooks")}
+                            </div>
+                          }
+                        >
+                          <div className="text-cool-gray-900 text-lg font-medium leading-7">
+                            {data?.totalNotebooks}
+                          </div>
+                        </DescriptionList.Item>
+                      </DescriptionList>
+                    </div>
+                  </div>
+                </div>
+                <div className="bg-cool-gray-50 px-5 py-3">
+                  <div className="text-sm leading-5">
+                    <Link
+                      href="/notebooks"
+                      className="font-medium text-blue-600 transition duration-150 ease-in-out hover:text-blue-900"
+                    >
+                      {t("View all")}
+                    </Link>
+                  </div>
+                </div>
+              </Block>
+              <Block>
+                <div className="p-5">
```

Comment:
Why not use `className` on Block directly ?

Reply:
Will update it.